test/parallel/test-runner-output.mjs

```diff
@@ -132,6 +132,15 @@ const tests = [
     name: 'test-runner/output/timeout_in_before_each_should_not_affect_further_tests.js',
     flags: ['--test-reporter=tap'],
   },
+  {
+    name: 'test-runner/output/test-timeout-flag.js',
+    flags: ['--test-reporter=tap'],
+  },
+  // --test-timeout should work with or without --test flag
+  {
+    name: 'test-runner/output/test-timeout-flag.js',
+    flags: ['--test-reporter=tap', '--test'],
```

Comment:
I took a quick look, so I might be wrong:
I think this is the first occurrence in the file where we use the same name twice (in this case, I assume the tests pass because the output is identical).
Was this intentional?

Reply:
Yes, this is intentional. This test ensures that the --test-timeout flag works regardless of whether the --test flag is present. As currently `--test-timeout` would not have any affect if `--test` flag is not present. So technically we are expecting the output to be identical.